api/exceptions.inc

```diff
@@ -3,6 +3,11 @@
 // Base exception that all API exceptions should derive from
 class ApiException extends Exception
 {
+    public function __construct($message = "API exception", $code = 0)
```

Comment:
Let's add a comment that these API error codes should be in range 0-99. Although we should perhaps use 1-99 to avoid some possible confusion since a return code of 0 on unix means success.

Reply:
done this